config/environment.rb

```diff
@@ -141,6 +141,22 @@ def render_facet_value(facet_field, item, options = {})
     end
   end
 
+  def render_selected_facet_value(facet_field, item)
+    # Before:
+    # content_tag(:span, :class => "facet-label") do
+    # content_tag(:span, facet_display_value(facet_field, item), :class => "selected") +
+    # # remove link
+    # link_to(content_tag(:span, '', :class => "glyphicon glyphicon-remove") + content_tag(:span, '[remove]', :class => 'sr-only'), search_action_path(remove_facet_params(facet_field, item, params)), :class=>"remove")
+    # end + render_facet_count(item.hits, :classes => ["selected"])
+    content_tag(:span, class: 'facet-label selected') do
+      link_to(
+        facet_display_value(facet_field, item),
+        search_action_path(remove_facet_params(facet_field, item, params)),
+        class: 'facet_select'
+      )
+    end
+  end
+
```

Comment:
@mccalluc one strategy for tracking changes of overridden Blacklight files may be to use once commit to bring the file in from the gem unchanged. Then use subsequent commits to change it. That way, it's all in our own git history, and then I'd be ok with having a single note: "This file comes from Blacklight gem /path/to/orig/file" instead of blocks of comments describing what was oveerwritten.

Just a suggestion.